### PR TITLE
Tb/ue5

### PR DESCRIPTION
**Pull request by toddbellMSFT**
Preliminary ue5 requirements.

### CODE REVIEW

**Review by pgilmorepf — APPROVED** [review]

**Comment by toddbellMSFT** [comment]
We can't add this until we have appropriate approval.

**Comment by NangiDev** [comment]
Any news about supporting UE5?

**Pull request closed without merging**
